README.md

```diff
@@ -11,4 +11,4 @@ Project to replace the old cut
 
 
 ## local setup for postgres running in docker 
- docker run --name postgres_cat -e POSTGRES_DB=cat -e POSTGRES_USER=cat -e  POSTGRES_PASSWORD=cat -d -p 5433:5432 postgres:12-alpine
\ No newline at end of file
+ docker run --name postgres_cat -e POSTGRES_DB=cat -e POSTGRES_USER=cat -e  POSTGRES_PASSWORD=cat -d -p 5433:5432 postgres:12-alpine
```

Comment:
I assume that these are test data sets.

Reply:
There are no (test/prod) data sets in the repo ... maybe we'll have to create a test data in the future ...